blue-note/templates/404.html

```diff
@@ -1,11 +1,19 @@
 <!-- wp:template-part {"slug":"header","tagName":"header"} /-->
 
-<!-- wp:group {"tagName":"main","layout":{"type":"constrained","justifyContent":"left"}} -->
-<main class="wp-block-group">
+<!-- wp:group {"tagName":"main","layout":{"type":"constrained","justifyContent":"left","contentSize":"1320px"}} -->
+<main class="wp-block-group"><!-- wp:heading {"level":1} -->
+<h1 class="wp-block-heading">This page could not be found.</h1>
+<!-- /wp:heading -->
 
-<!-- wp:pattern {"slug":"blue-note/hidden-404"} /-->
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"bottom":"60px","top":"59px"}}},"layout":{"type":"constrained","contentSize":"650px"}} -->
+<div class="wp-block-group alignfull" style="padding-top:59px;padding-bottom:60px"><!-- wp:paragraph {"style":{"spacing":{"padding":{"top":"0px"}},"typography":{"fontSize":"22px"}}} -->
+<p style="padding-top:0px;font-size:22px">The page you were looking for could not be found. It might have been removed, renamed, or did not exist in the first place.</p>
```

Comment:
As a follow up we should either remove the pattern or move this content to the pattern.